packages/react-ui/src/widgets/legend/LegendRamp.js

```diff
@@ -23,24 +33,53 @@ export default function LegendRamp({ isContinuous, legend }) {
 
   return (
     <Grid container item direction='column' spacing={1} data-testid='ramp-legend'>
-      <Grid container item>
-        {isContinuous ? (
-          <StepsContinuous palette={palette} />
-        ) : (
-          <StepsDiscontinuous labels={labels} palette={palette} max={max} min={min} />
-        )}
-      </Grid>
-      {/* <Tooltip title={'AVG: ' + avg} placement='top' arrow>
+      {error ? (
+        <Grid item className={classes.errorContainer}>
+          <Typography variant='overline'>
+            You need to specify valid numbers for the labels property
+          </Typography>
+        </Grid>
+      ) : (
+        <>
+          <Grid container item>
+            {isContinuous ? (
+              <StepsContinuous palette={palette} />
+            ) : (
+              <StepsDiscontinuous labels={labels} palette={palette} max={max} min={min} />
+            )}
+          </Grid>
+          {/* <Tooltip title={'AVG: ' + avg} placement='top' arrow>
         <Box className={classes.avg} style={{ left: `${avgPerc}%` }} />
       </Tooltip> */}
```

Comment:
Let's remove the commented code, right?

Reply:
please, every time you find, just delete

Reply:
Done.